index.html

```diff
@@ -387,6 +388,71 @@ <h2>Citation</h2>
     </div>
 </div>
 
+<!-- Contact Us -->
+<div class="container" style="max-width: 768px">
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Contact Us</h2>
+            <div class="row">
+                <div class="col-md-12">
+                    <form name="submit-to-google-sheet">
+                        <input type="text" name="Name" placeholder="Your Name" required>
+                        <input type="Email" name="Email" placeholder="Your Email" required>
+                        <textarea name="Message" rows="6" placeholder="Your Message"></textarea>
+                        <button type="submit" class="btn-secondary btn-secondary2">Submit</button>
+                    </form>
+                    <span id="msg"></span>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<script>
+var tablinks = document.getElementsByClassName("tab-links");
+var tabcontents = document.getElementsByClassName("tab-contents");
+
+function opentab(tabname) {
+    for (tablink of tablinks) {
+        tablink.classList.remove("active-link");
+    }
+    for (tabcontent of tabcontents) {
+        tabcontent.classList.remove("active-tab");
+    }
+    event.currentTarget.classList.add("active-link");
+    document.getElementById(tabname).classList.add("active-tab");
+}
+</script>
+<script>
+
+    var sidemenu = document.getElementById("sidemenu");
+
+    function openmenu() {
+        sidemenu.style.right = "0";
+    }
+    function closemenu() {
+        sidemenu.style.right = "-200px";
+    }
+
+</script>
+<script>
+    const scriptURL = 'https://script.google.com/macros/s/AKfycbwPg2KWzBAXWxwT8naOS0mzhoxwBZuQtwD9-SJBtlSS_4URxa8CPq-WUFoJMEq0daiClw/exec'
```

Comment:
Do we risk spam by making the url public? Are there any ways we can hide this, and still host through gh-pages?